src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs

```diff
@@ -242,5 +243,28 @@ private void CloseButton_Click(object sender, RoutedEventArgs e)
         {
             this.Close();
         }
+
+        private void ResetButton_Click(object sender, RoutedEventArgs e)
+        {
+            var overlay = App.Overlay;
+            MainWindowSettingsModel settings = ((App)Application.Current).MainWindowSettings;
+
+            if (overlay.CurrentDataContext is LayoutModel model)
+            {
+                model.IsSelected = false;
+                model.IsApplied = false;
+            }
+
+            overlay.CurrentLayoutSettings.ZonesetUuid = settings.BlankModel.Uuid;
+            overlay.CurrentLayoutSettings.Type = LayoutType.Blank;
+            overlay.CurrentDataContext = settings.BlankModel;
+
+            App.FancyZonesEditorIO.SerializeAppliedLayouts();
```

Comment:
This line is not needed as we serialize when Editor is closed and read from C++ only when we detect that Editor is closed. So this is just double serialization

Reply:
What about multi monitor? Is this applied while is still open or also in that case it's only applied after the editor is closed?

Reply:
At the moment we don't "listen" to the tmp file changes, so every change (apply/reset layout) in C++ happens only on closing Editor

Reply:
Exactly, until editor is closed, nothing will be introduced in the C++, so this is an obsolete call.

Reply:
OK, thanks.

Reply:
On closing event there is serialization only for the deleted layouts. Applied (and reset) layouts are serialized exactly when the `Apply`/`Reset` button was clicked. So there is no double serialization.
However, we can move applied layouts serialization to the `OnClosing` to get rid of code duplication.

Reply:
Ah, right. Apply was followed by Close before. Now it is not a case :) Yes, we can serialize on closing to have single source of trouth. Current approach is also fine, should not be a blocker for this PR